Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use Alpine-node as a base image because it is much smaller.
+#
+# Base image size for mhart/alpine-node:4 is 13 MB
+# Base image for standard node:4 is 253 MB.
+#
+# Our production deploy does not have any C/C++ library dependencies yet
+# so this is still easy to do.
+FROM mhart/alpine-node:4
```

Comment:
Also thinking about this more. It might be good to use a CentOS derived image. We've been flipping back and forth between distros, and CentOS-based ones are what all the current Ansible scripts are based on so that we have consistency on what to expect from both host and container OSes.